### ISSUE TITLE
Error in included file doesn't abort playbook run

### ISSUE THREAD

**Issue by bretmartin** [ansible/ansible#12210]
##### Issue Type:
- Bug Report
##### Ansible Version:

```
ansible 2.0.0 (devel e8d7fafc52) last updated 2015/09/02 09:17:49 (GMT -400)
  lib/ansible/modules/core: (detached HEAD bbcfb1092a) last updated 2015/09/02 09:56:03 (GMT -400)
  lib/ansible/modules/extras: (detached HEAD b8803306d1) last updated 2015/09/02 09:55:47 (GMT -400)
  config file = 
  configured module search path = None
```
##### Ansible Configuration:

`No config file found; using defaults`
##### Environment:

N/A but I am running on Mac OS X
##### Summary:

A failed task, even in an included file, should abort execution, but as of the rev noted above, it just skips the rest of that included file, and playbook execution continues.
##### Steps To Reproduce:

`roles/test/tasks/main.yml`:

```

---

- include: included.yml
- debug: msg="got here"
```

`roles/test/tasks/included.yml`:

```

---

- command: /usr/bin/false
- debug: msg="got here in included file"
```
##### Expected Results:

Execution should stop after the `false` task in `included.yml`
##### Actual Results:

```
ansible-playbook --vault-password-file=~/.ansible_vault_password \
        playbook-test.yml --limit=localhost

PLAY ***************************************************************************

TASK [setup] *******************************************************************
ok: [localhost]

TASK [test : include] **********************************************************
included: included.yml for localhost

TASK [test : command] **********************************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["/usr/bin/false"], "delta": "0:00:00.003706", "end": "2015-09-02 10:48:57.579885", "failed": true, "rc": 1, "start": "2015-09-02 10:48:57.576179", "stderr": "", "stdout": "", "stdout_lines": [], "warnings": []}

TASK [test : debug msg=got here] ***********************************************
ok: [localhost] => {
    "changed": false, 
    "msg": "got here"
}

PLAY RECAP *********************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=1   
```

**Comment by bretmartin** [comment]
@jimi-c Cc:ing you as requested on IRC

**Comment by drybjed** [comment]
I can confirm this behaviour in different production roles.

**Comment by jimi-c** [comment]
First attempt at fixing here, tested by @drybjed but not sure if it 100% solved his case.

**Comment by bretmartin** [comment]
@jimi-c Confirmed to fix my test case above. Thanks!

**Comment by jimi-c** [comment]
Excellent, thanks!

**Comment by ymattw** [comment]
FYI: this fix does not work when multiple `include:` is in use in a task, final fix in commit 9d61a6c works.
